Lib/fontTools/feaLib/builder.py

```diff
@@ -638,6 +642,12 @@ def buildGDEFMarkGlyphSetsDef_(self):
             sets.append(glyphs)
         return otl.buildMarkGlyphSetsDef(sets, self.glyphMap)
 
+    def buildDebg(self):
+        if not "Debg" in self.font:
+            self.font["Debg"] = newTable("Debg")
+            self.font["Debg"].data = {}
+        self.font["Debg"].data["io.github.fonttools.feaLib"] = self.lookup_locations
```

Comment:
in ufoLib we use "com.github.fonttools.ufoLib" as the fileCreator:

https://github.com/fonttools/fonttools/blob/097b255e68181b52bf74e3556e6d1b5fee653b2e/Lib/fontTools/ufoLib/__init__.py#L887